misc.php

```diff
@@ -7,6 +7,11 @@
 License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 */
 
+add_filter( 'upload_mimes', function( $mimes ) {
+	unset( $mimes['flv'] );
+	return $mimes;
+});
```

Comment:
Yeah they could always 1-up us - though us setting a high priority is also a signal to devs that this is meant to be the final say and not overridden, and makes an override more deliberate and perhaps easier to detect.